compiler/il/OMRDataTypes.hpp

```diff
@@ -229,19 +241,23 @@ enum DataTypes
    Float,
    Double,
    Address,
-   VectorInt8,
-   VectorInt16,
-   VectorInt32,
-   VectorInt64,
-   VectorFloat,
-   VectorDouble,
    Aggregate,
    NumOMRTypes,
 #include "il/DataTypesEnum.hpp"
-   NumTypes
+   NumNonVectorTypes,
```

Comment:
"Scalar" is a more conventional term to use here than "NonVector"

Reply:
I wanted to bring attention to the existence of the new vector types. But we can rename to scalar if that's more preferable.

Reply:
"Scalar" is the more preferable term to me.  Others may have different opinions.

Reply:
Sure. Will make the change soon.